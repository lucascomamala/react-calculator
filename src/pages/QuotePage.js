import React from 'react';

const QuotePage = () => (
  <blockquote>
    “Do not worry about your difficulties in mathematics. I can assure you mine are still greater.
    ”
    <br />
    – Albert Einstein
  </blockquote>
);

export default QuotePage;
