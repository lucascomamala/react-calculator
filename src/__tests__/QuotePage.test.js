import React from 'react';
import renderer from 'react-test-renderer';
import QuotePage from '../pages/QuotePage';

it('renders correctly', () => {
  const tree = renderer.create(<QuotePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
