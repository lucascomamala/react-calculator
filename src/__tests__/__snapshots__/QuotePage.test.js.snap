// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  hello from quote page
</div>
`;
